src/userInterface/textUserInterFace/InputHandler.java

```diff
@@ -0,0 +1,288 @@
+package userInterface.textUserInterFace;

```

Comment:
Okay so in the new input you separated the commands and movements. Just wondering why is it necessary to separate them?

Reply:
I might not be completely understanding this, but I noticed you created createDisplayAttackOptionsInstruction() and createCancelInstruction(). I know we do need a createDisplayAttackOptionsInstruction() for basic movement for players, just wondering what createCancelInstruction() really does here since.